modules/perception/conf/perception_lowcost_vis.conf

```diff
@@ -181,8 +181,9 @@
 
 --a_matrix_covariance_coeffcient_1=0.05
 --a_matrix_covariance_coeffcient_2=0.05
---dag_config_path=conf/dag_camera_obstacle_lane_motion_vis.config
```

Comment:
@techoe You can add a corresponding script under ./script for easier switching configs later